sysdeps/sigma/include/abi-bits/abi.h

```diff
@@ -0,0 +1 @@
+../../../../abis/sigma/abi.h
```

Comment:
This still links abis/sigma instead of abis/mlibc, even though both files are identical.

Reply:
my bad